apps/examples/webserver/webserver_main.c

```diff
@@ -111,25 +111,27 @@ static int http_send_chunk(struct http_client_t *client)
 	memset(msg_body, 'a', MAX_DATA_SIZE);
 	msg_body[MAX_DATA_SIZE] = 0;
 
-	ret = http_send_response_chunk(client, 200, "OK", msg_body, NULL, FIRST_DATA);
+	ret = http_send_response_chunk(client, 200, "OK", msg_body, MAX_DATA_SIZE, NULL, FIRST_DATA);
 	if (ret < 0) {
 		printf("Error: Fail to send FIRST_DATA \n");
 		return ret;
 	}
 
 	for (int i = 0; i < MAX_DATA_COUNT; i++) {
-		ret = http_send_response_chunk(client, 200, "OK", msg_body, NULL, NEXT_DATA);
+		ret = http_send_response_chunk(client, 200, "OK", msg_body, MAX_DATA_SIZE, NULL, NEXT_DATA);
```

Comment:
For status, constant value '200' is not good, in my opinion. is is general value of http like a 0666 or 0777 of file permission?

Reply:
done